src/interfaces/address-interfaces.ts

```diff
@@ -1,23 +1,34 @@
-// Base address Interface to match the address.json schema and it's optional fields.
+/** Address interface to match the address.json schema plus optional fields. */
 export interface AddressIF {
-  actions?: string[];
-  addressCity: string;
-  addressCountry: string;
-  addressRegion: string;
-  deliveryInstructions?: string;
-  postalCode: string;
-  streetAddress: string;
-  streetAddressAdditional?: string;
+  actions?: string[]
+  addressCity: string
+  addressCountry: string
+  addressRegion: string
+  deliveryInstructions?: string
+  postalCode: string
+  streetAddress: string
+  streetAddressAdditional?: string
+  addressType?: 'mailing' | 'delivery'
 }
 
-// Interface to define the joint base address response
-export interface BaseAddressObjIF {
```

Comment:
This name suggests it's the address used for "base address component", which is incorrect. So I renamed it per its current usage. (It's not used for director addresses atm.)